controllers/managedfusionoffering_controller.go

```diff
@@ -56,6 +56,7 @@ type ManagedFusionOfferingReconciler struct {
 //+kubebuilder:rbac:groups=misf.ibm.com,resources={managedfusionofferings,managedfusionofferings/finalizers},verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=misf.ibm.com,resources=managedfusionofferings/status,verbs=get;update;patch
 //+kubebuilder:rbac:groups="operators.coreos.com",resources={subscriptions,operatorgroups,catalogsources},verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups="network.openshift.io",resources=egressnetworkpolicies,verbs=create;get;list;watch;update
```

Comment:
Does it make sense to have this comment here instead of the `datafoundation.go` file. Do we want to assume that all offerings will need RBAC for `egressnetworkpolicies`?

Reply:
Updated